source/src/main/kotlin/org/neo4j/connectors/kafka/source/Neo4jCdcTask.kt

```diff
@@ -67,6 +67,7 @@ class Neo4jCdcTask : SourceTask() {
         CDCClient(
             config.driver,
             { sessionConfig },
+            { config.txConfig() },
```

Comment:
if the returned object doesn't change over invocations, we can pre-construct it and return that object to reduce object instantiations, like what we did for `sessionConfig`.